src/backend/web/routes/posts.js

```diff
@@ -21,6 +43,21 @@ posts.get('/', async (req, res) => {
     return;
   }
 
+  /**
+   * Add prev, next, first and last in the response's header.
+   * It's been implemented to work circularly.
+   * Once reached the last set of posts, 'next' points at the first set.
+   * Same case with 'prev' and the first set of posts.
+   */
+  const nextPage = to >= postsInDB ? 1 : parseInt(page, 10) + 1;
```

Comment:
Let's move this `parseInt()` call earlier to line 10 when you define it, so that it's always a number in the lines above, and here.